src/Search/Workflow/ResearchArticleWorkflow.php

```diff
@@ -132,7 +135,11 @@ public function index(ArticleVersion $article) : array
             'value' => json_encode($articleObject->dataSets ?? '[]'),
         ];
 
-        $sortDate = $article->getStatusDate();
```

Comment:
seems we don't use other dates (like `versionDate`), so this should be the only entry point for articles